xwiki-pro-macros-ui/src/main/resources/Confluence/Macros/RecentlyUpdated.xml

```diff
@@ -0,0 +1,1744 @@
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </supportsInlineMode>
+      <visibility>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <freeText>forbidden</freeText>
+        <largeStorage>0</largeStorage>
+        <multiSelect>0</multiSelect>
+        <name>visibility</name>
+        <number>6</number>
+        <prettyName>Macro visibility</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator>|</separator>
+        <separators>|</separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>Current User|Current Wiki|Global</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </visibility>
+    </class>
+    <property>
+      <async_cached>0</async_cached>
+    </property>
+    <property>
+      <async_context/>
+    </property>
+    <property>
+      <async_enabled>0</async_enabled>
+    </property>
+    <property>
+      <code>
+{{include reference="Confluence.Macros.RecentlyUpdatedService" /}}
+
+{{velocity output="false"}}
+## -----------------------------------------------------------------
+## -----------------------------------------------------------------
+## -----------------------------------------------------------------
+## Parameter formatting macros
+## -----------------------------------------------------------------
+
+## Format serialized comma-separated list coming from macro parameters
+## so that it can be used inside a solr filter query parameter
+## * Escape spaces inside values by wrapping them inside quotes,
+##   making sure "+" and "-" filter modifiers stay outside the quotes
+## * Remove commas so that values are separated only by spaces
+#macro (formatSerializedList $serializedList $return)
+  ## Wrap each value around quotes and replace commas by spaces
+  ## (adds extra quotes / spaces at the beginning and end of the string)
+  #set ($searchRegex = '(?:^|$|,)\s*([-+]?)')
+  #set ($replacement = '" $1"')
+  #set ($formatted = $!stringtool.replaceAll($serializedList, $searchRegex, $replacement))
+  ## Remove extra quotes / spaces at the beginning and end of the string
+  #set ($formatted = $!stringtool.substring($formatted, 2, -2))
+  ## Return
+  #set ($return = $NULL)
+  #setVariable("$return" $formatted)
+#end
+
+## Format serialized comma-separated author list coming from macro parameters
+## Author format is different between comments and document metadata
+## ("xwiki:" wiki prefix is missing from comments author metadata),
+## so add both versions in the formatted serialized list
+#macro (formatSerializedAuthorList $serializedAuthorList $return)
+  #set ($authorsList = [])
+  #set ($splittedAuthorList = $!stringtool.split($serializedAuthorList, ','))
+  #foreach ($author in $splittedAuthorList)
+    #set ($author = $author.strip())
+    #set ($searchRegex = '^([-+])?')
+    #set ($replacement = '$1xwiki:')
```

Comment:
Hard-coding the wiki name and making assumptions about the reference syntax is not nice. You need to resolve the author reference using the current resolver (i.e. relative to the current wiki).